manifests/init.pp

```diff
@@ -44,6 +44,8 @@
   $gid          = undef,
   $shell        = '/bin/true',
 
+  $no_websudo   = false,
```

Comment:
The Jira documentation says it is this: "Disabling Secure Administrator Sessions"; why not make it `$disable_secure_admin_sessions = false`?